### PR TITLE
Adds quotes around post

### LINKED ISSUE #163: @route("blah", method=POST) POST is undefined
I'd like to make a route in my plugins that allows a user to post data, but it says POST is undefined. If I try to post to the url without it, I get a 405 Method Not Allowed. Even putting POST in quotes and then trying, I still get the 405 Method Not Allowed

Following this example:
http://skoczen.github.io/will/plugins/notice/#handle-webhooks-and-web-pages

EDIT: This was mostly just me being stupid. POST definitely needs to be in quotes and then it's good to go.

### PR DESCRIPTION
**Pull request by brandonsturgeon**
resolves #163

### CODE REVIEW

**Comment by skoczen** [comment]
Awesome, thanks for this - will get it in on my next sprint!

File: docs/plugins/notice.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ def keep_alive(self):
     return {}
 
 # With full control, multiple templates, still connected to chat.
-@route("/complex_page/<page_id:int>", method=POST)
+@route("/complex_page/<page_id:int>", method='POST')
```
Comment (shadow7412) [inline docs/plugins/notice.md]: Double quotes are used everywhere else from what I can tell... Should this follow that convention?

**Comment by brandonsturgeon** [comment]
Thanks @shadow7412 , I changed it up to follow the pattern.

**Comment by skoczen** [comment]
Hey, just tossing out a quick note on this and all the other PRs to say thanks for your patience.  

Neither will nor your contributions have been forgotten!

We've just launched our app at my day job, and given that I'm currently the entirely of the dev team, finding cycles for open-source work hasn't been possible.  Things are settling down though, and I should have a proper block of time at the end of next sprint (end of September.)  Thanks for hanging in there!

**Comment by skoczen** [comment]
This is released in [0.9.3](http://skoczen.github.io/will/improve/#releases) - thank you so much for your patience, and even more so for your contributions that made will even better!

-Steven

**Comment by brandonsturgeon** [comment]
Hey @skoczen 

Thank you so much for getting stuff going! Always awesome to see improvements happening.

**Pull request merged**